nwb_conversion_tools/datainterfaces/ecephys/axona/axonadatainterface.py

```diff
@@ -124,121 +135,233 @@ def get_metadata(self):
                 for group_name in unique_elec_group_names
             ],
             Electrodes=[
-                dict(
-                    name="group_name",
-                    description="""The name of the ElectrodeGroup this electrode
-                                is a part of.""",
-                    data=[f"Group{x}" for x in elec_group_names],
-                )
+                dict(name="group_name", description="The name of the ElectrodeGroup this electrode is a part of.")
```

Comment:
```suggestion
```

We don't need this, group_name is already a required column of the electrodes table

Reply:
@sbuergers it looks like you never committed the suggestion, you just marked it as resolved.

Reply:
@bendichter this confuses me, I did indeed not commit the suggestion here in github, but changed in manually in vscode. The changes are also seen here: https://github.com/catalystneuro/nwb-conversion-tools/blob/axona_fixes/nwb_conversion_tools/datainterfaces/ecephys/axona/axonadatainterface.py

I am not sure why they do not show up here though. There is an 'outdated' tag next to the axonadatinterface link above this conversation.